src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -188,10 +188,6 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
                 //check if our contact lookup field is incorrect and update it
                 if (thisOpp.Primary_Contact__c != cr.ContactId) {
                     
-                    //update existing opp for OPP_OpportunityNaming call
```

Comment:
why did you add this last pull request, and why now must it go?  I was assuming it was needed to keep thisOpp in sync.